SignalMessaging/Models/OWSQuotedReplyModel.m

```diff
@@ -170,7 +170,8 @@ + (nullable OWSQuotedReplyModel *)quotedReplyForMessage:(TSMessage *)message
 
     if (!hasText && !hasAttachment) {
         OWSFail(@"%@ quoted message has neither text nor attachment", self.logTag);
-        return nil;
+        quotedText = @"";
+        hasText = YES;
```

Comment:
In prod, I suspect we should treat this as an "empty" message.  Note that this can happen if, for example, all of the quoted text is filtered.

Reply:
👍